templates/zerver/help/only-allow-admins-to-invite-new-users.md

```diff
@@ -0,0 +1,13 @@
+# Only allow admins to invite new users.
+By enabling this will only allow administrators to invite new users within
+the organisation. This can only be configured from an Administrator's account.
```

Comment:
Something needs to change about this sentence. First of all, you say "By enabling **this** will ...". What is **this**? And also, "By enabling this will" isn't grammatically correct I think. You could change the sentence to something like: "If you are an administrator, you can change a setting to only allow administrators to invite new users. This guide will teach you how to do this." That sentence isn't the best either, but it is grammatically correct.